index.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta name="description" content="UMass Amherst Webring" />
+    <title>UMass Amherst Web Ring</title>
+    <link rel="stylesheet" href="/styles.css" />
+    <style>
+      @import url("https://fonts.googleapis.com/css2?family=Forum&family=IBM+Plex+Mono:wght@300;400;500;600;700&display=swap");
+    </style>
+  </head>
+  <body>
+    <header><h1>UMass Amherst Web Ring</h1></header>
+    <main>
+      <div id="memberlist">
+        <ul id="list"></ul>
+      </div>
+    </main>
+    <hr />
+    <footer>
+      <a id="addsite" href="https://github.com/umacabal/umaring/pulls"
```

Comment:
```suggestion
      <a id="addsite" href="https://github.com/umacabal/umaring/"
```
I think just bringing people to the readme makes more sense since it includes instructions